### PR TITLE
Enum extensions

### LINKED ISSUE #11: Enum extensions
On the latest version of Dart SDK, it is possible to define extensions on `Enum` types.
Would be neat to see `enum_value.humanized` property on them.
above property converts enum value to string without the type name.

Sample:
```dart
extension EnumExtensions on Enum {
  String get humanized => toString().split('.').last;
}
```

### PR DESCRIPTION
**Pull request by ArunPrakashG**
added `enum.humanized` property to print enums as human readable form

closes #11

### CODE REVIEW

**Comment by kentcb** [comment]
Thanks @ArunPrakashG. Can you please make these changes:

- call into the [symbol name humanization](https://github.com/kentcb/humanizer/blob/master/doc/symbols.md) logic to humanize the enum name. e.g. instead of humanizing to `"applePie"` it would humanize to "apple pie"
- rename the property to `toHumanizedName()` (a method rather than property) for consistency

File: lib/src/transformations/enum_extensions.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// Provides extensions for [Enum] type
+extension EnumExtensions on Enum {
+  /// Returns an human readable value of this enum value.
```
Comment (kentcb) [inline lib/src/transformations/enum_extensions.dart:3]: Nit: please correct the grammar here to "a human-readable"

Comment (ArunPrakashG) [reply lib/src/transformations/enum_extensions.dart]: Hey, i have updated with those changes

**Review by kentcb — APPROVED** [review]

**Comment by kentcb** [comment]
Great, thanks for your contribution @ArunPrakashG!

**Pull request merged**